static/app/components/charts/baseChart.tsx

```diff
@@ -333,11 +340,11 @@ function BaseChartUnwrapped({
   onFinished,
   onRendered,
 
-  options = {},
-  series = [],
-  additionalSeries = [],
-  yAxis = {},
-  xAxis = {},
+  options = DEFAULT_OPTIONS,
+  series = DEFAULT_SERIES,
+  additionalSeries = DEFAULT_ADDITIONAL_SERIES,
+  yAxis = DEFAULT_Y_AXIS,
+  xAxis = DEFAULT_X_AXIS,
```

Comment:
Oh, does this really help? this isn't justinitialized once?

Reply:
Afaik these are just fn calls, so the references will be created fresh on each function call. Consider these example

old:
![CleanShot 2023-11-21 at 15 59 25@2x](https://github.com/getsentry/sentry/assets/9317857/3f64d75a-eb3f-49a6-94cf-9453937eca57)

new - stable default (not that mismatch is only called once)
![CleanShot 2023-11-21 at 16 00 37@2x](https://github.com/getsentry/sentry/assets/9317857/ba6af5ad-7b29-4755-b60c-fa672e76fc53)

The issue is that when you do what I did, you want to make these are never mutated (marked as readonly at least) so that they don't end up being mutated and your props don't start bleeding across component instances